static/css/styles.css

```diff
@@ -0,0 +1,5 @@
+body {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
```

Comment:
add empty line and more styles please

Reply:
Besides styles in style.css file I added also inline styles